bluehex.html

```diff
@@ -0,0 +1,263 @@
+<link rel="stylesheet" type="text/css"
+href="https://fonts.googleapis.com/css?family=Pacifico">
+<link rel="stylesheet" type="text/css"
+href="https://fonts.googleapis.com/css?family=Amatic+SC">
+
+</head>
+<body>
+  <div class = "container">
+    <div class = "row">
+      <center><div style="font-family: 'Pacifico', serif; font-size: 60px;">Complete the day</div>
+        <br>
+      </div>
+      <div class = "row">
+        <div style="font-family: 'Amatic SC', serif; font-size: 40px;">The percentage number is the amount of work you've done. The time limit is eight hours. Complete the assignment(reach 100%) before the day ends.</div>
+        <br>
+      </div></center>
+      <br>
+      <div class = "row">
+        <div class = "col-md-9">
+          <div class = "board">
+            <div id = "wrapper" style = "background-color:">
+
+            </div>
+          </div>
+        </div>
+        <div class = "col-md-3">
+          <div id = "scoreboard">
+            <h2>your color: </h2>
+            <div class = "box player"></div>
+            <br>
+            <div class = "text">
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </body>
+  <script>
+  var x = 0;
+  for(let i = 0; i < 10; i++){
+    $("#wrapper").append("<div class='row" + i + "'>");
+    for(let x = 0; x < 10; x++){
+      $("#wrapper").append("<div class = 'box' id='" + i + "-" + x + "'></div>")
+    }
+  }
+  let column = 0;
+  let row = 0;
+  //random blocks
+  for(let i = 0; i <= 25; i++){
+    let rand = Math.floor(Math.random() * (10)) + 0;
+    let rand2 = Math.floor(Math.random() * (10)) + 0;
+    $("#" + rand + "-" + rand2).addClass('block');
+  }
+  //random energies
+  var array = []
+
+  for(let i = 0; i <= 11; i++){
+    let rgb = "rgb(" + Math.floor(Math.random() * (255)) + ',' + Math.floor(Math.random() * (255)) + ','  + Math.floor(Math.random() * (255)) + ")"
+    array.push(rgb);
+  }
+  console.log(array);
+  for(let i = 0; i <= 11; i++){
+    let rand = Math.floor(Math.random() * (10)) + 0;
+    let rand2 = Math.floor(Math.random() * (10)) + 0;
+    $("#" + rand + "-" + rand2).attr("style", `background: ${array[i]}`).addClass('energy').removeClass('block');
+    console.log(rand + " " + rand2);
+  }
+  $('#' + row + '-' + column).addClass('player').removeClass('block');
+  arrayNum = 0
+  $("#scoreboard").append("<br><br><div><h2>color you need: </h2><div class = 'box colorNeed' style = 'background-color:" + array[arrayNum] +"'</div></div>");
+  function energyCollect (arrayNum, array, row, column){
+  console.log(arrayNum);
+  arrayNum = arrayNum + 1;
+  console.log(arrayNum);
+  $(".box.colorNeed").attr("style", `background: ${array[arrayNum]}`);
+  $('#' + row + '-' + column).removeClass('energy').attr("style", `background: black`);
+  }
+  document.onkeydown = function myFunction() {
+    switch (event.keyCode) {
+      //up key
+      case 38:
+      //checks if player is on top row
```

Comment:
Notice how you have a logic in this case statement. You can group the code into a function.

```
function upArrowToggled() {
  // does stuff
}
```